### DISCUSSION TITLE
Production

### DISCUSSION THREAD

**Discussion by americodls** [unfazedEgnimadevspace/Gracker#40] [Ideas]
@unfazedEgnimadevspace do you have any aspirations to lead this project to production?

I suggested this type of app because it could be useful for me.

I don't know if you hit the iron sometimes, but as a gym goer, I have some small issues that I would like to have an app to help me out.

Some of them:
* My memory is very bad so having an app to keep that info is very helpful to me. I register my exercises, I register my sessions, and I can use the app to recall every time I need. I've been solving that problem by keeping notes of my workouts in my google keep, which is not so great.
* Progress. I would love to see some graphs showing my progress on each exercise over time. Currently, I don't have any for that since I just have some not-well-structured notes on my phone.
* It would be nice to know for how long I am doing each exercise so that I can rotate and start stimulating my muscles in a different way. I have been doing that but blindly.

I believe that by using the app we can find lots of issues and design solutions for them.

So I am saying that I believe that is important to have something being used by someone to have an experience closer to the market reality.

What do you think? Do you want to move this project to production?

**Comment by unfazedEgnimadevspace** [comment]
Hello @americodls 

Definitely I want to push this app to production, I know we are still quite far from a production ready app but i would definitely love to have some users using it to solve some of their problems you just mentioned 

I have taken in to account the problems you stated here and they can features in the app in the future.
